nemo/export/vllm/model_loader.py

```diff
@@ -68,6 +68,9 @@ def _load_nemo_checkpoint_state(nemo_file: str):
 
         return sharded_state_dict
 
+    def download_model(self, model_config: ModelConfig) -> None:
+        raise NotImplementedError
```

Comment:
Needed for vllm >= 0.6.1.post1 https://github.com/vllm-project/vllm/blob/v0.6.1.post1/vllm/model_executor/model_loader/loader.py#L190